spec/browsers/apple_watch_user_agent_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'user_agent'
+
+shared_examples 'Apple Watch' do
+  it "returns 'Apple Watch' as its browser" do
+    expect(useragent.browser).to eq('Apple Watch')
+  end
+
+  it "returns 'WatchOS' as its operating system" do
+    expect(useragent.os).to eq('WatchOS')
+  end
+
+  it "returns 'Apple Watch' as its platform" do
+    expect(useragent.platform).to eq('Apple Watch')
+  end
+end
+
+describe "UserAgent: (null)/(null) watchOS/5.0 model/Watch2 ,7 hwp/t8002 build/16R349 (6; dt:149)" do
```

Comment:
instead of repeating all of this, I'd do something like 
```:ruby
{ 
  'UserAgent: (null)/(null) watchOS/5.0 model/Watch2 ,7 hwp/t8002 build/16R349 (6; dt:149)' => '5.0', 
  ...
}.each do |ua, version|
  context 'with #{ua} as user agent string' do
    let(:useragent) { UserAgent.parse(ua) }

    it_behaves_like 'Apple Watch'
    it { expect(useragent.version).to eql(version) }
  end
end
```